api/jsonrpc/eth_endpoint.go

```diff
@@ -138,12 +141,319 @@ func (e *Eth) GetBalance(address string, number string) (interface{}, error) {
 	return new(types.Big), nil
 }
 
+// GetStorageAt returns the contract storage at the index position
+func (e *Eth) GetStorageAt(address string, index []byte, number string) (interface{}, error) {
```

Comment:
- index []byte should be index types.Hash because the types.Hash object already has functions to UnmarshalJSON, otherwise, this function would fail.
- Number is a BlockNumber object

Reply:
Fixed 👍